core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -38,7 +38,8 @@ oppia.directive('collectionSummaryTile', [
         isPlaylistTile: '&isPlaylistTile',
         showLearnerDashboardIconsIfPossible: (
           '&showLearnerDashboardIconsIfPossible'),
-        isContainerNarrow: '&containerIsNarrow'
+        isContainerNarrow: '&containerIsNarrow',
+        belongsToCreatorPlaylist: '&belongsToCreatorPlaylist',
```

Comment:
Why "creator" playlist? I thought that the playlist was for the use of the learner.

Reply:
Yeah this could be misleading. How about ```belongsToCreator``` or ```belongsToCreatorDashboard```?

Reply:
How about: `isOwnedByCurrentUser '&activityIsOwnedByCurrentUser'`?

Reply:
Sounds good :+1: